ambry-rest/src/main/java/com/github/ambry/rest/StorageServerNettyChannelInitializer.java

```diff
@@ -93,6 +96,7 @@ protected void initChannel(SocketChannel ch) throws Exception {
     int peerPort = peerAddress.getPort();
     SslHandler sslHandler = new SslHandler(sslFactory.createSSLEngine(peerHost, peerPort, SSLFactory.Mode.SERVER));
     pipeline.addLast("SslHandler", sslHandler);
+    pipeline.addLast("securityChecker", this.serverSecurityChecker);
```

Comment:
`this` is not needed here. Align with other.

Reply:
fixed it.